incubator/barbarian/barbarian/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Barbarian is a big data system founded on Apache Hadoop offering Hive, LLAP and Tez for fast SQL datawarehousing at scale.
+name: barbarian
+version: 0.1.2-badh0.1.0
+icon: https://barbarians.io/barbarians.png
+maintainers:
+- name: rob gibbon
```

Comment:
Same here.

Reply:
changed to use github username, corrected indentation